### ISSUE TITLE
Separate game lobby from play table

### ISSUE THREAD

**Issue by brine** [octgn/OCTGN#1800]
The game lobby needs to be its own window, with the lobby players being their own identity outside of the play table players.

**Comment by kellyelton** [comment]
The serverside needs to be updated to reflect that change as well.

**Comment by brine** [comment]
yeah I imagine it'd be a big undertaking but I think it would add a lot of unique functionality

**Comment by kellyelton** [comment]
Yeah I agree. That was just a note for when someone comes along and wants to do this.
